### PR TITLE
Notificationsetting, password change, auth styling, Activities

### PR DESCRIPTION
**Pull request by dimitriirtimid**
Had a productive weekend ;)

This PR implements a number of issues:
- https://github.com/local-motion/product/issues/403: Change password: implemented and styled
image

Also restyled all other authentication functions:
image
image
image
image
image

- https://github.com/local-motion/product/issues/402 Notification optin/out: Checkbox is now connected
image
Note that I did not use/implement the save/cancel buttons and I propose not to do so. (Perhaps for UI reason switch from the checkbox to an on/off slide switch?)

- https://github.com/local-motion/product/issues/401: Serve playground membership for user profile working and (thus) connected to the backend
image

and finally:
- https://github.com/local-motion/product/issues/405 event history / activities
Not completed yet. Mainly all messages need to be defined and some further sorting/merging issues.
Also question for @maxvisser : How do you want the messages from various initiatives merged?
image

### CODE REVIEW

File: src/components/Activities/Activities.jsx [hunk 1]
```diff
@@ -47,8 +63,24 @@ const styles = theme => ({
 });
 
 class Activities extends Component {
+
+    getMessageForEventType = record => {
+        const {classes} = this.props
+        switch (record.eventType) {
+            case 'INITIATIVE_JOINED':
+                return record.actorName + ' doet lekker mee'
```
Comment (maxvisser) [inline src/components/Activities/Activities.jsx]: doet mee aan de actie

Comment (dimitriirtimid) [reply src/components/Activities/Activities.jsx]: :)
It was meant as a joke. The activities part is not finished yet. Need to define all messages (ao).

**Comment by dimitriirtimid** [comment]
Audit trail integration from various initiatives is now working. Also all messages are defined:
image

**Comment by dimitriirtimid** [comment]
badge is still a todo

**Comment by dimitriirtimid** [comment]
Added the notification badge:
image

It counts the notifications displayed in the list (max 20) since the last time the user looked at the notifications, excluding the events that were caused by the user himself.

**Comment by dimitriirtimid** [comment]
Added dynamic polling frequency: The frequency of the polling decreases when there are no new/modified results.

**Comment by dimitriirtimid** [comment]
Notification snackbar when the network connection is lost:
image

**Comment by dimitriirtimid** [comment]
Cognito tokens are refreshed each 20 mins to avoid the 'unauthenticated user' messages. When tokens can no longer be refreshed the user get a message : 'Je sessie is verlopen. Log opnieuw in'

**Comment by maxvisser** [comment]
Lets merge this. As it is getting pretty big feature packed pr :) Works great

**Pull request merged**